### PR TITLE
Update README in preparation for GA

### PR DESCRIPTION
**Pull request by jhaaaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,7 +255,7 @@ This package currently does not support message content compression.
 
 Because `xmtp-flutter` is in active development, you should expect breaking revisions that might require you to adopt the latest SDK release to enable your app to continue working as expected.
 
-XMTP communicates about breaking revisions in the [XMTP Discord community](https://discord.gg/xmtp), providing as much advance notice as possible. Additionally, breaking revisions in an `xmtp-flutter` release are described on the [Releases page](https://github.com/xmtp/xmtp-flutter/releases).
+XMTP communicates about breaking revisions in the [XMTP Discord community](https://discord.gg/xmtp), providing as much advance notice as possible. Additionally, breaking revisions in an `xmtp-flutter` release will be described on the [Releases page](https://github.com/xmtp/xmtp-flutter/releases).
```
Comment (jhaaaa) [inline README.md:258]: True about the Releases page?

File: README.md [hunk 2]
```diff
@@ -171,8 +168,8 @@ var listening = client.streamMessages(convo).listen((message) {
 await listening.cancel();
 ```
 
-**Note:** This package does not currently include the `streamAllMessages()` functionality from the XMTP
-client SDK for JavaScript (xmtp-js).
+> **Note**  
+> This package does not currently include the `streamAllMessages()` functionality from the [XMTP client SDK for JavaScript](https://github.com/xmtp/xmtp-js) (xmtp-js).
```
Comment (jhaaaa) [inline README.md:172]: Still true, correct @dmccartney? Just making sure. =)

Comment (jhaaaa) [reply README.md:172]: @yash-luna ... look what I found: https://github.com/community/community/discussions/16925. Closest thing we're going to get to a callout in GitHub README markdown from what I can tell. WDYT? <3

Comment (dmccartney) [reply README.md:172]: Confirmed!

**Review by dmccartney — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jhaaaa) [reply README.md:258]: I think it is safe to keep this as-is. We can iterate as the direction we need to take becomes clearer.

File: README.md [hunk 3]
```diff
@@ -21,6 +18,9 @@ To learn more about XMTP and get answers to frequently asked questions, see [FAQ
 
 For a basic demonstration of the core concepts and capabilities of the `xmtp-flutter` client SDK, see the [Example app project](https://github.com/xmtp/xmtp-flutter/tree/main/example).
 
+> **Important**  
+> This example app includes experimental caching functionality that has been found to degrade the performance of the example app. The cause of this performance issue is isolated to this caching functionality and the performance issue impacts the example app only. The `xmtp-flutter` client SDK itself is not the cause of and is not impacted by this performance issue.
+
```
Comment (dmccartney) [inline README.md:23]: suggestion:
> The example app includes a demonstration of how you might approach caching (offline storage). But beware: the demo app naively performs a full refresh very frequently -- this causes slowdowns. The underlying `xmtp-flutter` client SDK itself has no performance issues. But app developers wanting offline storage will need to design their refresh-strategies for performance. Future versions aim to make this aspect easier to manage.

File: example/README.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 An XMTP example app.
 
+> **Important**  
+> This example app includes experimental caching functionality that has been found to degrade the performance of the example app. The cause of this performance issue is isolated to this caching functionality and the performance issue impacts the example app only. The `xmtp-flutter` client SDK itself is not the cause of and is not impacted by this performance issue.
+
```
Comment (dmccartney) [inline example/README.md:7]: ditto suggestion above ^

**Review by dmccartney — CHANGES_REQUESTED** [review]

**Review by yash-luna — COMMENTED** [review]
How are we surfacing the typedocs in xmtp.org? 

There was discussion around how the docs won't be on .org but we'll provide a link to the docs and allow searching via Algolia

File: README.md (hunk 3 above)
Comment (jhaaaa) [reply README.md:23]: @dmccartney Thank you for this - how does my implementation above look to you? Just made a few tech writer nit edits. <3

File: example/README.md (hunk 4 above)
Comment (jhaaaa) [reply example/README.md:7]: @dmccartney Thank you for this - how does my implementation above look to you? Just made a few tech writer nit edits. <3

**Review by dmccartney — APPROVED** [review]

**Comment by jhaaaa** [comment]
@yash-luna I know we discussed in slack, but sharing here as well if others are interested. 

Here is a PR of the doc updates that we'll make on xmtp.org to surface the xmtp-flutter quickstart and reference docs: https://github.com/xmtp/xmtp-dot-org/pull/262

**Pull request merged**